component/src/main/resources/installation-scripts/install.bat

```diff
@@ -17,19 +17,49 @@ rem specific language governing permissions and limitations
 rem under the License.
 rem
 
-SETLOCAL
-SET /P ballerina_home=Please enter Ballerina home:
+setlocal
 
-IF NOT EXIST "%ballerina_home%/bin/ballerina.bat" (
-    ECHO "Incorrect Ballerina Home provided!"
+IF NOT EXIST "%BALLERINA_HOME%/bin/ballerina.bat" (
+    SET /P ballerina_home=Couldn't find Ballerina home in your system!.Please enter Ballerina home:
+)
+
+IF NOT EXIST "%BALLERINA_HOME%/bin/ballerina.bat" (
```

Comment:
Shouldn't this be %ballerina_home% ?

Reply:
When we will install the ballerina using .msi, Ballerina home will be set itself into the system's environment with name as BALLERINA_HOME.